contracts/test/InitiativeUniV4Merkl.t.sol

```diff
@@ -0,0 +1,115 @@
+        console2.log(governance.getLatestVotingThreshold(), "voting threshold");
+        (Governance.VoteSnapshot memory voteSnapshot, Governance.InitiativeVoteSnapshot memory initiativeVoteSnapshot) =
+            governance.snapshotVotesForInitiative(address(uniV4MerklRewardsInitiative));
+        console2.log(initiativeVoteSnapshot.votes, "initiative Votes");
+        (Governance.InitiativeStatus status, uint256 lastEpochClaim, uint256 claimableAmount) =
+            governance.getInitiativeState(address(uniV4MerklRewardsInitiative));
+        console2.log(uint256(status), "uint(status)");
+        console2.log(lastEpochClaim, "lastEpochClaim");
+        console2.log(claimableAmount, "claimableAmount");
+        */
+
+        // Claim for initiative
+        (,, uint256 claimableAmount) = governance.getInitiativeState(address(uniV4MerklRewardsInitiative));
+        // Creating a campaign is expensive, and uses more than the allowed by Gorvernace contract, so we need a wrapper
+        // governance.claimForInitiative(address(uniV4MerklRewardsInitiative));
+        uniV4MerklRewardsInitiative.claimForInitiative();
+        //console2.log(boldToken.balanceOf(address(uniV4MerklRewardsInitiative)), "boldToken.balanceOf(address(uniV4MerklRewardsInitiative))");
+        assertEq(boldToken.balanceOf(address(uniV4MerklRewardsInitiative)), 0, "Initiative should have sent incentives to campaign");
+        // Check campaign
+        uint256 epochEnd = EPOCH_START + (governance.epoch() - 1) * EPOCH_DURATION;
+        IDistributionCreator.CampaignParameters memory params = IDistributionCreator.CampaignParameters({
+            campaignId: bytes32(0),
+            creator: address(uniV4MerklRewardsInitiative),
+            rewardToken: address(boldToken),
+            amount: claimableAmount,
+            campaignType: uniV4MerklRewardsInitiative.CAMPAIGN_TYPE(),
+            startTimestamp: uint32(epochEnd),
+            duration: uint32(EPOCH_DURATION),
+            campaignData: uniV4MerklRewardsInitiative.getCampaignData()
+        });
+        bytes32 campaignId = merklDistributionCreator.campaignId(params);
+        IDistributionCreator.CampaignParameters memory campaign = merklDistributionCreator.campaign(campaignId);
+        assertEq(campaign.creator, params.creator, "creator");
+        assertEq(campaign.rewardToken, params.rewardToken, "rewardToken");
+        assertEq(campaign.amount, params.amount * 97 / 100, "amount minus fees");
+        assertEq(campaign.campaignType, params.campaignType, "campaignType");
+        assertEq(campaign.startTimestamp, params.startTimestamp, "startTimestamp");
+        assertEq(campaign.duration, params.duration, "duration");
+        assertEq(campaign.campaignData, params.campaignData, "campaignData");
+
+        assertGt(
+            boldToken.balanceOf(address(merklDistributionCreator.distributor())),
+            0,
```

Comment:
Earlier, we assert that the entire BOLD is gone from `uniV4MerklRewardsInitiative`. Is this where it goes (minus fees)? If so, can we assert exactly how much should be here?

Reply:
Yes, fixing it.